### PR TITLE
Move from OC_Log_XX to \OC\Log

### PR DESCRIPTION
**Pull request by rullzer**
CC: @nickvergessen @DeepDiver1975 @MorrisJobke @LukasReschke

### CODE REVIEW

File: core/Command/Log/Manage.php [hunk 1]
```diff
@@ -115,7 +115,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 	 * @throws \InvalidArgumentException
 	 */
 	protected function validateBackend($backend) {
-		if (!class_exists('OC_Log_'.$backend)) {
+		if (!class_exists('OC\\Log\\'.ucfirst($backend))) {
```
Comment (nickvergessen) [inline core/Command/Log/Manage.php]: ucfirst wont work for `ErrorHandler`, so I guess you can remove this and we have to require people putting in the right camelcase as well

Comment (rullzer) [inline core/Command/Log/Manage.php]: But it is `Errorlog` right? (I guess that is why it was also originally named that way)

Comment (nickvergessen) [inline core/Command/Log/Manage.php]: Right, sorry for the confusion

**Comment by nickvergessen** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**